baseball/src/App.test.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {cleanup, render, fireEvent} from 'react-testing-library';
+import 'jest-dom/extend-expect';
+
+import App from './App';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App />, div);
+  ReactDOM.unmountComponentAtNode(div);
+});
+
+describe('Baseball', () => {
+  afterEach(cleanup);
+
+  describe('BUTTON FUNCTIONS', () => {
+    describe('Strike Button', () => {
+      it('should increase strikes display by 1', () => {
+        const {getByText} = render(<App/>);
+        expect(getByText(/strikes: 0/i));
+        fireEvent.click(getByText('Strike'));
+        expect(getByText(/strikes: 1/i));
+      });
+  
+      it('should rollover strikes display to 0 after 3', () => {
+        const {getByText,debug} = render(<App/>);
+        debug();
+        expect(getByText(/strikes: 0/i));
+        fireEvent.click(getByText('Strike'));
+        fireEvent.click(getByText('Strike'));
+        expect(getByText(/strikes: 2/i));
+        fireEvent.click(getByText('Strike'));
+        expect(getByText(/strikes: 0/i));
```

Comment:
this is the ideal way of testing a button that changes displayed data like this--verifying default state, then verifying at multiple places along the way before the final version which should be the default state again